src/tree/v2/application/ApplicationResourceGroupingManager.ts

```diff
@@ -141,11 +142,24 @@ export class ApplicationResourceGroupingManager extends vscode.Disposable {
     }
 
     private groupByResourceType(context: ResourceGroupsTreeContext, resources: ApplicationResource[]): GroupingItem[] {
+        const initialGrouping = {};
+
+        // Pre-populate the initial grouping with the supported resource types...
+        azureExtensions.forEach(extension => {
+            extension.resourceTypes.forEach(resourceType => {
+                initialGrouping[resourceType] = [];
+            });
+        });
+
+        // Exclude resource groups...
+        resources = resources.filter(resource => resource.azureResourceType.type !== 'microsoft.resources/resourcegroups');
```

Comment:
Don't we have an `AzExtResourceType` for this? If not we should add one.

Reply:
I guess it depends on whether we want to treat resource groups as first-class Azure resources.  The existing tree view does not.

Reply:
I don't think we have to treat them as first-class while still assigning them a resource type

Reply:
Ok, added an [issue](https://github.com/microsoft/vscode-azuretools/issues/1244) for them to be added to the enumeration.